puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/rpm_packaging.groovy

```diff
@@ -1,5 +1,8 @@
 def packages_to_build
 def packages = [:]
+def VERCMP_NEWER = 12
```

Comment:
Should these be defined in `rpm_packaging.groovy` so every job that uses this can use them?

Reply:
If this functionality were re-factored out I'd lean towards yes, but for now keeping them close to the code that uses them.